index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
```

Comment:
Kindly I suggest you use polite and clear comments, like:

```
This is a polite suggestion: "I think it would be better to use semantic tags"

This reads as an order: "Use semantic tags"
```